packages/css/src/components/button/README.md

```diff
@@ -2,16 +2,16 @@
 
 # Button
 
-Allows the user to perform actions and operate the user interface.
+Allows the user to perform an action or operate the interface.
 
 ## Guidelines
 
-- Choose a short label that describes the function of the button.
-- Use the correct type of button for the corresponding application, e.g. `type="submit"` for the primary form button.
-- Make sure one can operate a button through a keyboard.
+- Describe the Button’s function through a short label in the infinitive form.
+- Use only 1 primary Button per screen.
 - Wrap 2 or more consecutive buttons and/or links in an [Action Group](https://designsystem.amsterdam/?path=/docs/components-layout-action-group--docs).
-- Primary, secondary and tertiary buttons can be used side by side.
-  Skipping levels is allowed.
+- Add `type="submit"` to make the Button submit a form.
+- Do not use a Button for navigation.
+  Use e.g. a [Standalone Link](https://designsystem.amsterdam/?path=/docs/components-navigation-standalone-link--docs) instead.
```

Comment:
There is a bit of grey area here. You can use a button to submit a form, and redirect when that's done. That is sort of using a button for navigation.

Reply:
```suggestion
- Do not use a Button to link to another page.
  Use a [Standalone Link](https://designsystem.amsterdam/?path=/docs/components-navigation-standalone-link--docs) instead, or an inline [Link](https://designsystem.amsterdam/?path=/docs/components-navigation-link--docs) in the running text.
```